src/status_im2/contexts/syncing/how_to_pair/view.cljs

```diff
@@ -0,0 +1,131 @@
+(ns status-im2.contexts.syncing.how-to-pair.view
+  (:require
+    [status-im2.contexts.syncing.how-to-pair.style :as style]
+    [status-im2.common.resources :as resources]
+    [utils.i18n :as i18n]
+    [quo2.core :as quo]
+    [quo2.foundations.colors :as colors]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]))
+
+(defn list-icon
+  [s]
+  [rn/view style/list-icon
```

Comment:
Style should be set in `:style` prop, but this is related to my comment above https://github.com/status-im/status-mobile/pull/15710/files#r1173260760

Reply:
e.g `[rn/view {:style style/list-icon} ...`

Reply:
Thanks @J-Son89! I moved the component props to the view file and refactored the view to reduce duplication of properties, leaving styles in style.cljs.